scripts/clog.inputs

```diff
@@ -31,7 +31,7 @@
 ../src/platform/pcp.c
 ../src/platform/storage_winuser.c
 ../src/platform/storage_posix.c
-../src/platform/crypt_openssl.c
+../src/platform/crypt_quictls.c
```

Comment:
I think we should keep this as `crypt_openssl.c` since quictls doesn't fork the crypto layer.

Reply:
yeah, I'm not sure how I let that in there.  I think it was just part of the mass rename of openssl->quictls.  Will update to reflect the right file name